src/metabase/driver/mysql.clj

```diff
@@ -160,6 +160,7 @@
 
 (defn- date-format [format-str expr] (hsql/call :date_format expr (hx/literal format-str)))
 (defn- str-to-date [format-str expr] (hsql/call :str_to_date expr (hx/literal format-str)))
+(defn- sql-cast [expr newtype] (hsql/call :cast expr (hsql/raw (name newtype))))
```

Comment:
This already exists as `metabase.util.honeysql-extansions/cast`

Reply:
fixed!